app/lib/pages/homework/homework_dialog.dart

```diff
@@ -223,12 +223,27 @@ class _SaveButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return IconButton(
-      icon: const Text("SPEICHERN",
-          style: TextStyle(fontWeight: FontWeight.w600, color: Colors.white)),
-      tooltip: "Hausaufgabe speichern",
-      iconSize: 90,
-      onPressed: () => onPressed(context),
+    return Tooltip(
+      message: "Hausaufgabe speichern",
+      child: ElevatedButton(
```

Comment:
If we want we to share the code of the buttons, not sure where to put the file for that and how to name it. Or we leave it like that.

Reply:
I mean it would make sense to share it because we mostly have the same format for the save/edit layouts for homework, blackboard and maybe events in the future.   
What about `/lib/sharezone_widgets`?